src/datamigration/azext_datamigration/generated/custom.py

```diff
@@ -77,11 +152,10 @@ def datamigration_sql_managed_instance_cancel(client,
                                               resource_group_name,
                                               managed_instance_name,
                                               target_db_name,
-                                              migration_operation_id=None,
+                                              migration_operation_id,
                                               no_wait=False):
     parameters = {}
-    if migration_operation_id is not None:
-        parameters['migration_operation_id'] = migration_operation_id
+    parameters['migration_operation_id'] = migration_operation_id
```

Comment:
Is this also a breaking change that `--migration-operation-id` is required now for `az datamigration sql-managed-instance cancel`?

Reply:
Yes, actually if necessary to add this parameter even before but not displayed as a required one from CLI help message point view. We have updated that in these changes.  I will add this to history.rst.

Reply:
Done